styleguide/derek/pattern-components/headers/headers.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Zoolander Banner Module', () => {
+  it('is defined', () => {
+    expect(Zoolander.Banners).to.not.be.undefined;
+  });
+
+  describe('Banners', () => {
+    beforeEach(() => {
+      $('body').html('');
+    });
+
+    it('should adjust the left angle', () => {
+      $('body').append('<div class="refreshBanner"><div class="refreshBanner-container"><div class="refreshBanner-boxContainer"><div class="refreshBanner-textRow"><div class="refreshBanner-textContainer refreshBanner-textContainerInterior"><h2 class="refreshBanner-headline">we guarantee a 99.99% openstack API uptime.</h2></div></div><span class="refreshBanner-line right-bottom">&nbsp;</span></div><span class="refreshBanner-line left-top">&nbsp;</span></div></div>');
+      const offset = $('.refreshBanner-line.left-top').offset();
+      Zoolander.Banners.init();
+      setTimeout(() => {
+        expect(offset).to.not.eql($('.refreshBanner-line.left-top').offset());
+      }, 1000);
+    });
```

Comment:
I don't think we need all this fancy behavior in our unit tests. This tests should be written the same way we did the filter bar tests. 

See: https://github.com/rackerlabs/zoolander/blob/master/dist/js/global-components/filter-bar/filter-bar.spec.js

This may mean some slight refactoring of the code.   

What needs to be tested is the actual mathematical calculations.  If we can vouch for inputs and outputs in all situations, that's enough test coverage. The tests as currently written only test whether or not offsets are not equal to one another.  It doesn't matter if they are correct calculations or not, just that they aren't the same value. 

By doing the other method we can get rid of DOM manipulation, the need to initialize code mid test, and get rid of fittext related timeouts (which may or may not flake out depending on how flaky connections decide to be at any given moment).